testing/integration/upgrade_test.go

```diff
@@ -162,50 +181,68 @@ func TestStandaloneUpgrade(t *testing.T) {
 		Sudo:    true, // requires Agent installation
 	})
 
-	agentFixture, err := define.NewFixture(t, define.Version())
-
-	require.NoError(t, err)
-
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	err = agentFixture.Prepare(ctx)
-	require.NoError(t, err, "error preparing agent fixture")
 
-	err = agentFixture.Configure(ctx, []byte(fastWatcherCfg))
-	require.NoError(t, err, "error configuring agent fixture")
+	versionList := getUpgradableVersions(ctx, t, define.Version())
+
+	for _, parsedVersion := range versionList {
 
-	const minVersionString = "8.9.0-SNAPSHOT"
-	minVersion, _ := version.ParseVersion(minVersionString)
+		if parsedVersion.String() == define.Version() || parsedVersion.IsSnapshot() {
+			t.Logf("Skipping upgrade test for version %q: same version as destination: %v snapshot: %v",
+				parsedVersion, parsedVersion.String() == define.Version(), parsedVersion.IsSnapshot())
```

Comment:
It might be better to place this inside of a `t.Run` and use `t.Skip` inside of it. That would ensure that in the HTML report that its a clear test that was skipped.

Reply:
I just noticed that this is a leftover from when we iterated over all versions...
with the new behavior of `getUpgradableVersions()` the `if` condition is never true, I am gonna remove it